Cargo.toml

```
@@ -15,6 +15,8 @@ categories = ["rust-patterns"]
 license = "Apache-2.0 OR MIT OR Zlib"
 readme = "README.md"
 
+exclude = ["*.html", "*.sh"]
```

Indeed, I just tried a `0.2.1-rc1` release off this branch, and got an error: https://docs.rs/crate/macro_rules_attribute/0.2.1-rc1/builds/2185887

So we'll have to make do with that `.html`:

```suggestion
exclude = ["*.sh"]
```

  - At which point, we'll now need a `0.2.1-rc2` version; I can take care of that, but feel free to run `./change_version 0.2.1-rc2` yourself if you want, and trust the script (you can see it's just a `sed` invocation 🙂)

---

Thanks for confirming. I think everything should be right now, including updating the version.